src/test/java/org/broadinstitute/consent/http/authentication/AuthenticatorTest.java

```diff
@@ -131,17 +141,45 @@ void testAuthenticateGetUserWithStatusInfoIncompleteClaims() throws Exception {
    */
   @Test
   void testUnknownNameDefaultsToEmail() {
-    String bearerToken = RandomStringUtils.randomAlphabetic(100);
-    MultivaluedMap<String, String> headerMap = new MultivaluedHashMap<>();
     headerMap.put(ClaimsCache.OAUTH2_CLAIM_access_token, List.of(bearerToken));
     headerMap.put(ClaimsCache.OAUTH2_CLAIM_email, List.of("email"));
     headerMap.put(ClaimsCache.OAUTH2_CLAIM_name, List.of("unknown"));
     headerCache.loadCache(bearerToken, headerMap);
-    oAuthAuthenticator = new OAuthAuthenticator(samService);
 
     Optional<AuthUser> authUser = oAuthAuthenticator.authenticate(bearerToken);
-    assertTrue(authUser.isPresent());
-    assertEquals(authUser.get().getName(), authUser.get().getEmail());
+    assertEquals(authUser.orElseThrow().getName(), authUser.orElseThrow().getEmail());
+  }
+
+  @Test
+  void testAuthenticateGetUserInfoWithDuosUser() {
+    headerMap.put(ClaimsCache.OAUTH2_CLAIM_access_token, List.of(bearerToken));
+    headerMap.put(ClaimsCache.OAUTH2_CLAIM_email, List.of("email"));
+    headerMap.put(ClaimsCache.OAUTH2_CLAIM_name, List.of("name"));
+    headerCache.loadCache(bearerToken, headerMap);
+    when(userService.findUserByEmail(headerMap.get(ClaimsCache.OAUTH2_CLAIM_email).get(0))).thenReturn(new User());
+
+    AuthUser authUser = oAuthAuthenticator.authenticate(bearerToken).orElseThrow();
+    assertInstanceOf(DuosUser.class, authUser);
+    assertNotNull(((DuosUser) authUser).getUser());
```

Comment:
if you kept the `user` passed to `thenReturn()` you could use `assertEquals()` here.